apps/docs/docs/core/core.md

```diff
@@ -176,7 +176,24 @@ export const contract = c.router({
 });
 ```
 
-## Strict Response Status Codes
+## Options
+
+These configuration options allow you to modify how your contract functions.
+
+### Base Header
+
+You can assign `baseHeaders` which will be merged with the contract `headers`. Here's how to set it:
+
+```typescript
+const c = initContract();
+export const contract = c.router({
+  // ...endpoints
+}, {
+  strictStatusCodes: true,
+});
+```
```

Comment:
You said `heres how to set it` then didn't show how to set it :D

Reply:
ooooo Whooooops